src/ProcessSlave.php

```diff
@@ -322,7 +319,35 @@ function ($controller) {
                 $this->sendMessage($this->controller, 'register', ['pid' => getmypid(), 'port' => $port]);
             }
         );
+    }
+
+    /**
+     * Attempt a connection through the unix socket until it succeeds.
+     * This is a workaround for an issue where the (hardcoded) 1s socket timeout is triggered due to a busy socket.
+     */
+    private function tryConnect()
```

Comment:
What happens if this never succeeds?

Reply:
From the process manager's perspective, the worker would be stuck in a perpetual starting state as it does right now when the slaves crash to the unhandled exception, so w/r/t that, it's an improvement.

But would this ever happen? Is there a scenario in which it would?

Reply:
We might listen for process on exit and recycle the slave if that happens before a connection was established. Just an idea, maybe to complicated....